playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -66,6 +65,24 @@ class NameSilencerParams(ActionParams):
     names: List[str]
 
 
+class SilenceAlertParams(ActionParams):
+    """
+    :var log_silence: prints a log in robusta logs of the silence.
+    """
+
+    log_silence: bool = False
+
+
+@action
+def silence_alert(alert: PrometheusKubernetesAlert, params: SilenceAlertParams):
```

Comment:
@arikalon1 LMK if you can think of anything else to add here

Reply:
it's perfect